src/menuFolder/SettingMenu.py

```diff
@@ -0,0 +1,51 @@
+from src.interaction import SubMenu, header, main, urwid, settings
+from src.gamestate import gamestate
+import os.path
+import json
+def clamp(n, min, max):
+    if n < min:
+        return min
+    elif n > max:
+        return max
+    else:
+        return n
+
+class SettingMenu(SubMenu):
+    type = "SettingMenu"
+    setting_options = ["set sound volume"]
```

Comment:
please add a newline before each method. I know visual studio does it itself, but i think it should be there in real text, too